elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java

```diff
@@ -122,13 +158,36 @@ public static void main(String[] args) {
 
             DynamicConfigValidator dynamicConfigValidator = new DynamicConfigValidator(configDir);
             dynamicConfigValidator.readAndValidateConfigs();
+            System.out.println("Configs Validation Passed!");
+
+            if (cli.hasOption("nocompile")) {
+                System.out.println("Skipped compilation for both Model and DB configs");
+                System.exit(0);
+            }
+
+            if (cli.hasOption("nomodelcompile")) {
+                System.out.println("Skipped compilation for Model configs");
+            } else {
+                System.out.println("Compiling Model configs (Use '--nomodelcompile' to skip this step).");
+                dynamicConfigValidator.hydrateAndCompileModelConfigs();
+                System.out.println("Model Configs Compilation Passed!");
+            }
+
+            if (cli.hasOption("nodbcompile")) {
```

Comment:
What is the use case for compiling models but not compiling db configs?

Reply:
basically if connection to db is not possible then it can be useful.